Code/GraphMol/FilterCatalog/Wrap/rough_test.py

```diff
@@ -543,6 +543,41 @@ def testThreadedRunner(self):
     self.assertEquals(len(results[0]), 1)
     self.assertEquals(results[0][0].GetDescription(), "no valid RDKit molecule");
 
+  def testThreadedPythonFilter(self):
+
+    class MWFilter(FilterCatalog.FilterMatcher):
+
+      def __init__(self, minMw, maxMw):
+        FilterCatalog.FilterMatcher.__init__(self, "MW violation")
+        self.minMw = minMw
+        self.maxMw = maxMw
+
+      def IsValid(self):
+        return True
+
+      def HasMatch(self, mol):
+        mw = rdMolDescriptors.CalcExactMolWt(mol)
```

Comment:
The filter catalog runner was hanging on any call to the subclassed MWFilter.

This also checks the logging if python logging is enabled.